### PR TITLE
changes

### PR DESCRIPTION
**Pull request by yashkapoor8710**
Simulator Screen Shot - iPhone 11 - 2020-02-20 at 12 19 41
Simulator Screen Shot - iPhone 11 - 2020-02-20 at 12 19 52

### CODE REVIEW

File: pages/BaseNext/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        history.push('/')
+    }  
+
+    return (
+        <>
+        <Button
+          label="Back"
+          onButtonPress={goToBaseHome}
+        />
+        </>
```
Comment (manojsinghnegiwd) [inline pages/BaseNext/index.tsx]: You don't need a fragment here. Please remove fragment

File: config/DefaultConfig.ts [hunk 2]
```diff
@@ -49,13 +40,6 @@ export const lightTheme: AppTheme = {
...
     constants: {
-      dateFormats: {
-        chatDateFormat: "MM/DD/YY",
-        chatHourDisplayFormat: "hh:mm A",
-        chatYesterdayText: "Yesterday",
-      },
-      searchPlacerHolder: "Search Chats",
-      messagePlacerHolder: "Type Text here for message...",
-      chatTitle: "Base App"
+      chatTitle: "Base App",
```
Comment (manojsinghnegiwd) [inline config/DefaultConfig.ts]: Rename this to `title`

File: components/Base/ButtonItem.tsx [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
 }: Props) => {
     const theme: AppTheme = useTheme();
 
     return (
-        <TouchableOpacity onPress={onChatPress}>
+        <TouchableOpacity onPress={onButtonPress}>
+            <View>
+                <StatusBar
+                    translucent
+                    barStyle={theme ? "dark-content" : "light-content"}
```
Comment (manojsinghnegiwd) [inline components/Base/ButtonItem.tsx]: I don't understand the logic here @yashkapoor8710

Comment (yashkapoor8710) [reply components/Base/ButtonItem.tsx]: I have remove this

File: config/DefaultConfig.ts (hunk 2 above)
Comment (yashkapoor8710) [reply config/DefaultConfig.ts]: I have renamed it

File: pages/BaseNext/index.tsx (hunk 1 above)
Comment (yashkapoor8710) [reply pages/BaseNext/index.tsx]: I have removed this

**Pull request merged**